src/ui/widgets/common/WidgetForm.js

```diff
@@ -163,7 +177,7 @@ export class WidgetFormBody extends Component {
                   optionDisplayName="name"
                   defaultOptionId="app.chooseAnOption"
                 />
-                {(mode === 'edit' && parentWidget) && (
+                {((mode === 'edit' || mode === MODE_CLONE) && parentWidget) && (
```

Comment:
can you make a const var for 'edit'? e.g. MODE_EDIT. make sure to use this across the component file.

Reply:
Yes, will change that!